assets/scss/typography.scss

```diff
@@ -1,54 +0,0 @@
-@import url(https://fonts.googleapis.com/css2?family=PT+Sans:ital,wght@0,400;0,700;1,400;1,700&display=swap);
-
-$pt-sans: 'PT Sans', sans-serif;
-$fonts: (
-        'pt-sans' : $pt-sans
-);
-
-@each $name, $font in $fonts {
-  .primary-font--#{$name} {
-    font-family: $font;
-
-    //GovUK Font Override
-    .govuk-body,
-    .govuk-details,
-    .govuk-accordion__open-all,
-    .govuk-accordion__section-heading,
-    .govuk-accordion__section-button {
-      font-family: $font;
-    }
-  }
-}
-
-//PT Sans Specific font sizes - Font size maybe something we want to add to customizer
-.primary-font--pt-sans {
-  font-size: 100%;
-
-  @import 'fontstyles/pt-sans';
-}
```

Comment:
We will need to keep these styles. As these styles set the font sizes when using PT Sans for all the headings and elements in the theme. They were set to match the design.

Reply:
Ah - maybe.  I will look at this again.

Reply:
Yes, possible we do need to replace these font styles.  I was lumping this in with the block restyling rather than part of the framework, as the affected elements are all within blocks.  

However, I could tackle them at this level and would be happy to do so.

Reply:
I have restored this file - most of it is still needed as you said.  This was the "overzealous" cut that I mentioned this morning.  

However, a few of the styles are still redundant and these have been removed in my redone one.  I've also encased the styles inside
```
#primary{
}
```